.cursor/rules/stories-format.mdc

```diff
@@ -0,0 +1,465 @@
+    },
+    tags: ['autodocs', 'dev'],
+};
+```
+
+### Anatomy stories
+
+Stories demonstrating component structure:
+
+```typescript
+/**
+ * Description of the component's anatomy and key parts.
+ * Explain what makes up the component.
+ */
+export const Anatomy: Story = {
+    render: (args) => html`
+        ${template({ ...args /* specific anatomy example */ })}
+    `,
+    tags: ['anatomy'],
+    args: {},
+};
+```
+
+### States stories
+
+Stories demonstrating different states (if applicable):
+
+```typescript
+/**
+ * Description of this particular state.
+ */
+export const StateName: Story = {
+    render: (args) => html`
+        ${template({ ...args /* state example */ })}
+    `,
+    tags: ['states'],
+    args: {},
+};
+```
+
+### Options stories
+
+Stories demonstrating variants, sizes, styles:
+
+```typescript
+/**
+ * Description of this option/variant.
+ */
+export const OptionName: Story = {
+    render: () => html`
+        ${/* examples */}
+    `,
+    tags: ['options'],
+};
+```
+
+### Accessibility story
+
+Required accessibility demonstration:
+
+```typescript
+/**
+ * ### Features
+ *
+ * The `<sp-component-name>` element implements several accessibility features:
+ *
+ * 1. **Feature name**: Description
+ * 2. **Feature name**: Description
+ *
+ * ### Best Practices
+ *
+ * - Best practice one
+ * - Best practice two
+ */
+export const Accessibility: Story = {
+    render: () => html`
+        ${/* accessibility examples */}
+    `,
+    tags: ['a11y'],
+};
+```
+
+## Tag usage requirements
+
+### Required tags
+
+- `'autodocs'` - Only on Playground story
+- `'dev'` - Only on Playground story
+- `'anatomy'` - For anatomy stories
+- `'states'` - For state demonstration stories (if component has states)
+- `'options'` - For variant, size, and style stories
+- `'a11y'` - For accessibility demonstration story
+
+### Exclusion tags
+
+- `'!test'` - Exclude from test runs (e.g., complex static color stories)
+- `'!dev'` - Exclude from dev Storybook
+
+### Migration tag
+
+- `'migrated'` - On the meta object to indicate refined format
+
+## JSDoc requirements
+
+### Above each story export
+
+Every story export must have a JSDoc comment explaining:
+
+- What it demonstrates
+- Any important context or usage notes
+- Best practices if relevant
+
+**Exception**: Do NOT add JSDoc comments above the Playground story.
+
+Use markdown formatting within JSDoc:
+
+- `**Bold**` for emphasis
+- Bullet lists for multiple points
+- Code formatting with backticks
+
+Example:
+
+```typescript
+/**
+ * Semantic variants allow you to render the badge with a descriptive name that maps to a design-system-aligned color.
+ *
+ * Use these variants for the following statuses:
+ * - **Positive**: approved, complete, success, new, purchased, licensed
+ * - **Informative**: active, in use, live, published
+ * - **Negative**: error, alert, rejected, failed
+ */
+export const SemanticVariants: Story = {
+    // ...
+};
+```
+
```

Comment:
Should we include something about making the three sizes a single story?

Also should we do this for variants as well?